docs/user_guide.md

```diff
@@ -10,4 +10,51 @@ Cyclone provides a powerful workflow engine that serves general workflow purpose
 $ make run_examples SCENE=cicd REGISTRIES=<registry>/<project>
 ```
 
-In addition to YAML format manifest files, you can also create your workflows via APIs provided by Cyclone Server. Refer to [API Docs](./swagger-api-docs.md) for detailed APIs.
\ No newline at end of file
+In addition to YAML format manifest files, you can also create your workflows via APIs provided by Cyclone Server. Refer to [API Docs](./swagger-api-docs.md) for detailed APIs.
+
+## Integration
+
+Cyclone provides an integration center for users to integrate external system information, including sensitive information such as `username`, `password`, `token`, etc,
+and Cyclone will store them by Kubernetes `Secret`.
+
+Cyclone supports 5 types of integrations: `SCM`, `Cluster`, `DockerRegistry`, `SonarQube` and `General`, and SCM is subdivided into the following subclasses:
+`GitHub`, `GitLab`, `BitBucket` and `SVN`.
+
+Supported external systems including:
+
+|  Name  | Sub Class | Supported Versions | Note  |
+| :---:  | :---:     | :---:              | :---: |
+| SCM    | GitHub    | enterprise edition is unsupported |  |
+|        | GitLab    | \>= 8.13.6         | May [affect 10.6 and later versions](#GitLab-Webhooks) to create a SCM webhook automatically triggered pipeline |
+|        | BitBucket | \>= 5.0, BitBucket cloud is unsupported | Different edition may affect [BitBucket Token and Webhooks](#BitBucket-Token-and-Webhooks) |
```

Comment:
It'd be better to add `BitBucket cloud is unsupported` in `Note`.

Reply:
OK